checkstyle-checks/src/main/resources/com/sonatype/checks/packagenames.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Copyright (c) 2011-present Sonatype, Inc. All rights reserved.
+    Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+    "Sonatype" is a trademark of Sonatype, Inc.
+
+-->
+<!DOCTYPE checkstyle-packages PUBLIC
+    "-//Puppy Crawl//DTD Package Names 1.0//EN"
+    "http://www.puppycrawl.com/dtds/packages_1_0.dtd">
+
+<checkstyle-packages>
```

Comment:
How does this file get picked up by checkstyle?  I don't see anything else in this PR that specifically refers to it.

Reply:
> How does this file get picked up by checkstyle? I don't see anything else in this PR that specifically refers to it.

Honestly, I don't think it was being picked up.  I removed the file.